package.json

```diff
@@ -30,7 +30,12 @@
     "docs:show": "compodoc -p ./tsconfig.doc.json -s --disablePrivate --disableProtected --disableLifeCycleHooks --disableInternal",
     "docs:json": "compodoc -p ./tsconfig.doc.json -e json -d . --disablePrivate --disableProtected --disableLifeCycleHooks --disableInternal",
     "storybook": "npm run docs:json && start-storybook -p 6006",
-    "build-storybook": "npm run docs:json && build-storybook"
+    "build-storybook": "npm run docs:json && build-storybook",
+    "aws:build": "npm run build-storybook",
```

Comment:
Maybe have sense to add the word "storybook". Like "aws:build-storybook", "aws:deploy-storybook".